### ISSUE TITLE
[Feature Request] Option to disable NSFW tag reccomendations in the search bar.

### ISSUE THREAD

**Issue by da123rrell** [Jays2Kings/tachiyomiJ2K#856]
### Why/User Benefit/User Problem
I do not want my friends to think i am reading loli mangas and adult mangas. My friend saw it and he's been bugging me about it ever since, just becausr it said "Search "Loli" or "Adult""
 
No specific requirements, just that.

**Comment by GabenRulez** [comment]
I agree that such an option would be quite useful, but it might be hard to differentiate whether a manga is NSFW or not.

 For now, the extensions are labelled "18+" or not, but mangas aren't.
